apps/docs/src/styles/main.scss

```
@@ -16,19 +16,15 @@
-    --REM: clamp(0.95rem, 0.9rem + 0.2cqw, 1rem); /* lism-css の --REM の cqw 版 */
-    font-size: var(--REM);
-
-    --hl-unit: calc(var(--REM) * 0.125);
-    --hl--xs: var(--hl-unit);
-    --hl--s: calc(var(--hl-unit) * 2);
-    --hl--base: calc(var(--hl-unit) * 3);
-    --hl--l: calc(var(--hl-unit) * 4);
-
+    --REM: clamp(0.95rem, 0.9rem + 0.2cqw, 1rem); /* cqw でほんのり流体タイポグラフィ */
     --s-unit: calc(var(--REM) * 0.5);
+    font-size: var(--REM);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)